fix(apple): Add SentrySDK dependency with xCode (#4674)
getsentry/sentry-docs: src/platforms/apple/common/install/swift-package-manager.mdx

description: "Integrate Sentry into your Xcode project using Swift Package Manager (SPM)."
---

To integrate Sentry into your Xcode project using Swift Package Manager (SPM), open your App in Xcode and open `File` -> `Add Packages`. Then add the SDK by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git` in the top right search field. Define your dependency rule by selecting the SDK version (or branch), and then click the "Add Package" button.

> When using SPM version 5.2 or later and your project uses `Package.swift` file to manage dependencies, you must specify the the target name too:

Author: ```suggestion
When using SPM version 5.2 or later and your project uses `Package.swift` file to manage dependencies, you must specify the target name too:
```

```swift {tabTitle:Swift}
.package(name: "Sentry", url: "https://github.com/getsentry/sentry-cocoa", from: "{{ packages.version('sentry.cocoa') }}"),
```

<Note>

Version tags or branches need to have the Package.swift file in it or Xcode won't be able to install the package. This means versions previous to `4.4.3` *cannot* be installed using SPM.
 

>  </Note>

Reviewer: Since this is updated, I'd also change in src/docs/clients/cocoa/index.mdx

description: "Integrate Sentry into your Xcode project using Swift Package Manager (SPM)."
---

> To integrate Sentry into your Xcode project using Swift Package Manager (SPM), open your App in Xcode and open `File` -> `Add Packages`. Then add the SDK by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git` on the top-right search field. Define you dependecy rule by selecting the SDK version (or branch), then press "Add Package" button.

Reviewer 2: ```suggestion
To integrate Sentry into your Xcode project using Swift Package Manager (SPM), open your App in Xcode and open `File` -> `Add Packages`. Then add the SDK by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git` in the top right search field. Define your dependency rule by selecting the SDK version (or branch), and then click the "Add Package" button.
```

To integrate Sentry into your Xcode project using Swift Package Manager (SPM), open your App in Xcode and open `File` -> `Add Packages`. Then add the SDK by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git` on the top-right search field. Define you dependecy rule by selecting the SDK version (or branch), then press "Add Package" button.

> When using SPM version 5.2 or later, you must specify the the target name too:

Reviewer: Isn't this important anymore?

Author: There is no way of doing this anymore with Xcode

Reviewer: Is it still necessary when using older Xcode versions? if so, we should add a note that this is only for older Xcode versions and not remove it.

Author: Good point. And I believe is possible to open a older project in the new xCode.

> NOTE: Version tags or branches need to have the Package.swift file in it or Xcode won't be able to install the package. Thus versions previous to `4.4.3` can't be installed via SPM!

Reviewer 2: We avoid exclamation points in the docs, so I've spelled out the word "cannot" instead of having it be a contraction and bolded it to make it more prominent. Also I've styled this as a note. 

```suggestion
<Note>

Version tags or branches need to have the Package.swift file in it or Xcode won't be able to install the package. This means versions previous to `4.4.3` *cannot* be installed using SPM.
 
 </Note>
```
